vbfml/scripts/err_analysis

```diff
@@ -797,6 +798,10 @@ def plot_correl(ctx, input_dir: str) -> None:
 
         print(f"plot {featurename} ({count}/{len(features.keys())})")
 
+        if featurename == "dataset_label":
+            print("no correl for dataset_label")
```

Comment:
I think we can remove the print statement here, we can just internally skip this feature name.